targets/unity-v2/source/Client/Authentication/Strategies/EmailPasswordAuthStrategy.cs

```diff
@@ -26,8 +26,43 @@ public void Authenticate(PlayFabAuthService authService, Action<LoginResult> res
             {
                 Email = authService.Email,
                 Password = authService.Password,
-                InfoRequestParameters = authService.InfoRequestParams
-            }, resultCallback, errorCallback);
+                InfoRequestParameters = authService.InfoRequestParams,
+                TitleId = PlayFabSettings.TitleId
+            }, resultCallback, error =>
+            {
+                // If user not registered, create one.
+                // Do it because LoginWithEmailAddressRequest does not contains "CreateAccount" option.
+                if (error.Error == PlayFabErrorCode.AccountNotFound)
+                {
+                    UnityEngine.Debug.LogWarning("Authentication failed. Try to register account.");
+                    PlayFabClientAPI.RegisterPlayFabUser(new RegisterPlayFabUserRequest
+                    {
+                        Username = authService.Username,
+                        Password = authService.Password,
+                        Email = authService.Email,
+                        InfoRequestParameters = authService.InfoRequestParams,
+                        TitleId = PlayFabSettings.TitleId
+                    }, registerSuccess =>
+                    {
+                        // convert RegisterResult to LoginResult
+                        resultCallback.Invoke(new LoginResult
+                        {
+                            AuthenticationContext = registerSuccess.AuthenticationContext,
+                            EntityToken = registerSuccess.EntityToken,
+                            CustomData = registerSuccess.CustomData,
+                            NewlyCreated = true,
+                            PlayFabId = registerSuccess.PlayFabId,
+                            SessionTicket = registerSuccess.SessionTicket,
+                            SettingsForUser = registerSuccess.SettingsForUser,
+                            InfoResultPayload = new GetPlayerCombinedInfoResultPayload(),
```

Comment:
I would rather this just be null, since it's a problem with the API that this isn't returned.

Reply:
Ok, i will set it to null.